<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PluginDetailsWidgetClass</class>
 <widget class="QWidget" name="PluginDetailsWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PluginDetailsWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="labelPluginVersion">
     <property name="text">
      <string>Plugin Version</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelPluginName">
     <property name="text">
      <string>Plugin Name</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelPluginType">
     <property name="text">
      <string>Plugin Type</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
